mgmtsystem/security/mgmtsystem_security.xml

```diff
@@ -52,4 +52,94 @@
         <field name="domain_force">['|',('company_id','=',False),('company_id','child_of',[user.company_id.id])]</field>
     </record>
 
+    <!-- =================== -->
+    <record id="no_document_rule_mgmtsystem_viewer" model="ir.rule">
+        <field name="name">No document page for viewers</field>
+        <field name="model_id" ref="document_page.model_document_page"/>
+        <field name="domain_force" eval="[(0, '=', 1)]"/>
+        <field name="groups" eval="[(4, ref('mgmtsystem.group_mgmtsystem_viewer'))]"/>
+        <field name="perm_read" eval="True"/>
+        <field name="perm_write" eval="True"/>
+        <field name="perm_create" eval="True"/>
+        <field name="perm_unlink" eval="True"/>
+    </record>
+
+    <record id="no_document_history_rule_mgmtsystem_viewer" model="ir.rule">
+        <field name="name">No document page history for viewers</field>
+        <field name="model_id" ref="document_page.model_document_page_history"/>
+        <field name="domain_force" eval="[(0, '=', 1)]"/>
+        <field name="groups" eval="[(4, ref('mgmtsystem.group_mgmtsystem_viewer'))]"/>
+        <field name="perm_read" eval="True"/>
+        <field name="perm_write" eval="True"/>
+        <field name="perm_create" eval="True"/>
+        <field name="perm_unlink" eval="True"/>
+    </record>
+    <!-- =================== -->
+
+    <!-- =================== -->
+    <!-- Read access in 'Knowledge / Knowledge User' group for all documents -->
+    <record id="document_page_rule_document_user_r" model="ir.rule">
+        <field name="name">All document pages (read)</field>
+        <field name="model_id" ref="document_page.model_document_page"/>
+        <field name="domain_force">[(1, '=', 1)]</field>
+        <field name="groups" eval="[(4, ref('knowledge.group_document_user'))]"/>
+        <field name="perm_read" eval="True"/>
+        <field name="perm_write" eval="False"/>
+        <field name="perm_create" eval="False"/>
+        <field name="perm_unlink" eval="False"/>
+        <field name="active" eval="False"/>
```

Comment:
I did this thinking that if someone wanted to install only the mgmtsystem module and not document_page_work_instruction, document_page_procedure or any other that added a registration rule related to document page, it would not make sense to apply this rule, but on second thought, it doesn't make much sense that you only install this module, because it is a base module. So, i'm going to commit this suggestion.